drf_renderer_xlsx/renderers.py

```diff
@@ -304,6 +307,14 @@ def _append_item(key, value):
                 _append_item(new_key, v)
         return dict(items)
 
+    def _sanitize_value(self, raw_value):
+        # prepend ' if raw_value is starting with possible malicious char
+        if self.sanitize_fields and raw_value:
+            str_value = str(raw_value)
```

Comment:
Should this be sanitizing integer fields? I have a test failing on this release because an integer that was being returned is now a string.